internal/services/logic/logic_app_standard_data_source.go

```diff
@@ -316,6 +326,28 @@ func dataSourceLogicAppStandardRead(d *pluginsdk.ResourceData, meta interface{})
 		}
 	}
 
+	ftpBasicAuth, err := client.GetFtpAllowed(ctx, id)
+	if err != nil || ftpBasicAuth.Model == nil {
+		return fmt.Errorf("retrieving FTP publish basic authentication policy for %s: %+v", id, err)
+	}
+
+	if props := ftpBasicAuth.Model.Properties; props != nil {
+		if err := d.Set("ftp_publish_basic_authentication_enabled", props.Allow); err != nil {
```

Comment:
Because this is just a bool, we shouldn't need to check the error on setting this value. It's really only lists and sets that we want to check that error